packages/viz/src/webpack.config.js

```diff
@@ -41,14 +41,30 @@ module.exports = {
   },
   plugins: [
     new HtmlWebPackPlugin({
-      template: path.join(__dirname, '../src/index.html'),
+      filename: 'index.html.template',
       inject: false,
+      template: path.join(__dirname, '../src/templates/index.html.template'),
     }),
     new CopyPlugin({
       patterns: [
-        { from: './src/example-policies/', to: `${distPath}/` },
         { from: './src/assets/', to: `${distPath}/assets/` },
       ],
     }),
   ],
+  resolve: {
+    alias: {
+      'three-forcegraph': path.resolve(
+        __dirname,
+        '../../../node_modules/three-forcegraph/dist/three-forcegraph.js'
+      ),
+      'force-graph': path.resolve(
+        __dirname,
+        '../../../node_modules/force-graph/dist/force-graph.js'
+      ),
+      'react-force-graph-2d': path.resolve(
+        __dirname,
+        '../../../node_modules/react-force-graph-2d/dist/react-force-graph-2d.js'
+      ),
```

Comment:
this could use some comments as to why it's necessary.  easier to do this, assuming these are resolvable:

```suggestion
      'three-forcegraph': require.resolve('three-forcegraph/dist/three-forcegraph.js'),
      'force-graph': require.resolve('force-graph/dist/force-graph.js'),
      'react-force-graph-2d': require.resolve('react-force-graph-2d/dist/react-force-graph-2d.js'),
```

if that doesn't work, you will need to use `path.resolve` the same way, but omit directory seps for portability.  this is tedious, so maybe:

```suggestion
      'three-forcegraph': path.resolve(
        __dirname,
        ...['../../../node_modules/three-forcegraph/dist/three-forcegraph.js'].split('/')
      ),
      'force-graph': path.resolve(
        __dirname,
        ...['../../../node_modules/force-graph/dist/force-graph.js'].split('/')
      ),
      'react-force-graph-2d': path.resolve(
        __dirname,
        ...['../../../node_modules/react-force-graph-2d/dist/react-force-graph-2d.js'].split('/')
      ),
```

Reply:
If you want Windows support, you'll have to test for it yourself - can merge into this branch if you have something confirmed working. And `require.resolve` requires a little bit more. Added a comment, though!

(Keeping them as one line make them a lot more readable IMO but prettier disagrees)